src/providers/MeetingProvider/docs/MeetingProvider.stories.mdx

```diff
@@ -90,4 +100,13 @@ For more information on Simulcast, check Amazon Chime JS SDK [simulcast guide](h
 There may be use cases to re-use the same `MeetingManager` instance accross multiple different `MeetingProvider`s.
 Hence, you can create a new `MeetingManager` instance and then pass it as a prop to your `MeetingProvider`s. If not passed,
 a new `MeetingManager` instance will be created internally with each `MeetingProvider` when rendered; and, 
-you will get the `MeetingManager` instance associated with that particular `MeetingProvider` when `useMeetingManager` hook is called.
\ No newline at end of file
+you will get the `MeetingManager` instance associated with that particular `MeetingProvider` when `useMeetingManager` hook is called.
+
+This approach has limitations. This should be used only in very specific cases where you want to share the meeting manager instance
```

Comment:
Can we link to the above Note instead of copy it down here?

Reply:
As discussed we would need to add a link add-on to storybook to do this. So, will check it later.